cropper/src/main/java/com/theartofdev/edmodo/cropper/ImageCropperFileProvider.java

```diff
@@ -0,0 +1,10 @@
+package com.theartofdev.edmodo.cropper;
+
+import androidx.core.content.FileProvider;
+
+/**
+ * Dummy file provider
```

Comment:
It's not dummy. Include a better explanation, something like:

    /**
     * Library-specific subclass of [FileProvider] so that the original class can still be declared by the consuming app.
     * See https://commonsware.com/blog/2017/06/27/fileprovider-libraries.html for details.
     */